src/components/DashboardPanel/DashboardPanel.js

```diff
@@ -88,4 +89,6 @@ export default class DashboardPanel extends Component {
 
 DashboardPanel.propTypes = {
   selectedNodes: PropTypes.object,
```

Comment:
For Moment.js prop-types we should be using `PropTypes.instanceOf(Date)` instead of `PropTypes.object`, as it is more specific.